### DISCUSSION TITLE
[3차 스프린트 - 기획 회의] 기능 명세서 정의

### DISCUSSION THREAD

**Discussion by Minjoo522** [woowacourse-teams/2024-devel-up#184] [백로그 논의]
#### 논의가 필요한 내용

오늘(7월 30일(화)) 미션에서 사용자 시나리오를 정리하고 기능 명세서를 정의했습니다.
각자 담당한 기능 명세서를 최대한 구체화 해서 시트에 작성해 주세요!

코멘트로 달아야 하는 내용은 다음과 같습니다!
1. 각자 기능명세서 시트 링크
2. 전체적으로 반영이 필요하다고 생각하는 피드백
3. (쓰고싶으면) 느낀점이나 하고싶은 말 아무거나!

### 내일 회의 전까지 해야하는 숙제(?)는 다음과 같습니다!
1. 다른 팀원의 기능 명세서 읽기
2. 구체적인 피드백이 필요하다면 스프레드 시트에 코멘트 달기

**이슈를 판 이유는 전체적 피드백 작성과 문서화를 위해서 입니다!**

**Comment by Parkhanyoung** [comment]
[기능명세서 링크](https://docs.google.com/spreadsheets/d/1jqbEg_gfXRH2pEft-0WsdYDaAHvgb1HEcgM4trbjw_A/edit?gid=2144982297#gid=2144982297)

### 나누고 싶은 말
오늘은 어제에 비해 다들 기분이 좋아보이고, 합이 잘 맞았던 것 같아 기뻤습니다 😄 한편 오늘은 전반적인 방향을 결정하는 회의여서 다함께 논의를 하고 만장일치를 이끌어내는 방식이었던 것 같은데, 이로 인해 결정 하나하나를 내리는 데 시간과 에너지가 많이 들기도 했던 것 같습니다. 오늘은 그럴 수밖에 없었다고 생각합니다. 다만 스프린트 기간에 비해 일의 진척이 늦은 만큼, 이번 전체 논의 이후에 구체적 기획이나 디자인, 미션 제작 등의 일을 할 때에는 태스크를 나누어 가져가는 방식으로 빠르게 의사결정을 하면 좋겠다는 생각이 들었습니다!

+내일 3차 스프린트 요구사항 중 하나인 ```BE + FE 함께 개발```을 어떻게 처리해보면 좋을지 논의해봐요!

오늘 하루도 다들 정말 고생 많았습니다! 💪🏻

**Comment by le2sky** [comment]
[기능명세서 링크](https://docs.google.com/spreadsheets/d/1jqbEg_gfXRH2pEft-0WsdYDaAHvgb1HEcgM4trbjw_A/edit?gid=1326771864#gid=1326771864)

다들 오늘 하루도 고생 많으셨습니다. 👍

**Comment by Minjoo522** [comment]
[기능명세서 링크](https://docs.google.com/spreadsheets/d/1jqbEg_gfXRH2pEft-0WsdYDaAHvgb1HEcgM4trbjw_A/edit?gid=2014604315#gid=2014604315)

기획 회의 쉽지 않지만 다들 힘내봅시다 👍

**Comment by chosim-dvlpr** [comment]
[기능명세서 링크](https://docs.google.com/spreadsheets/d/1jqbEg_gfXRH2pEft-0WsdYDaAHvgb1HEcgM4trbjw_A/edit?gid=1030703716#gid=1030703716)

지난 스프린트의 기획보다 더 구체적으로 잡혀가고 있는 것 같아서 좋았습니다. 시행착오를 통해 우리가 더 성장하고 있음을 느낄 수 있었어요!
조금만 더 힘내서 3차 스프린트를 잘 끝내보아요🍀

**Comment by alstn113** [comment]
[기능명세서](https://docs.google.com/spreadsheets/d/1jqbEg_gfXRH2pEft-0WsdYDaAHvgb1HEcgM4trbjw_A/edit?gid=709799636#gid=709799636)

**Comment by lilychoibb** [comment]
[기능명세서](https://docs.google.com/spreadsheets/d/1jqbEg_gfXRH2pEft-0WsdYDaAHvgb1HEcgM4trbjw_A/edit?gid=1668051534#gid=1668051534)

**Comment by brgndyy** [comment]
[기능명세서](https://docs.google.com/spreadsheets/d/1jqbEg_gfXRH2pEft-0WsdYDaAHvgb1HEcgM4trbjw_A/edit?gid=880304491#gid=880304491)

**Comment by robinjoon** [comment]
[링크](https://docs.google.com/spreadsheets/d/1jqbEg_gfXRH2pEft-0WsdYDaAHvgb1HEcgM4trbjw_A/edit?gid=525027947#gid=525027947)
